composer.json

```diff
@@ -14,10 +14,11 @@
 	},
 	"require-dev": {
 		"dealerdirect/phpcodesniffer-composer-installer": "^0.7.0",
-		"squizlabs/php_codesniffer": "3.6.0",
+		"squizlabs/php_codesniffer": "^3.7.1",
```

Comment:
I'm curious why PHPCS was pinned back on 3.6.0. The current version is 3.7.2. @jrfnl I'm sure you know!

Reply:
I'd definitely recommend an update, but until WPCS 3.0.0 is released the point is moot.

PHPCS 3.6.1 - 3.7.2 do contain some useful bugfixes, but the majority of updates are related to supporting new PHP syntaxes, which WP Core cannot use anyway since the minimum PHP version is 5.6, so for WP the benefit of updating would be minimal.

Also see the changelogs: https://github.com/squizlabs/PHP_CodeSniffer/releases

It does need to be pinned to a specific version as there is no committed `composer.lock` file and having PHPCS be a moving target which can fail builds at random whenever a new version is tagged is not desirable, so using `^` in the version constraint is out of the question.

Aside from that, I'd recommend an update to PHPCS to be done in a separate PR as it may throw new warnings/errors/allow for ignore annotations to be removed, which should not be mixed in with this PR.